dali/test/python/operator/test_coord_flip.py

```diff
@@ -91,5 +94,5 @@ def test_operator_coord_flip():
                 layout_shape_values.append(("xy", (0, 2)))
             for layout, shape in layout_shape_values:
                 for center_x, center_y, center_z in [(0.5, 0.5, 0.5), (0.0, 1.0, -0.5)]:
-                    yield check_operator_coord_flip, \
-                          device, batch_size, layout, shape, center_x, center_y, center_z
+                    yield check_operator_coord_flip, device, batch_size, layout, \
+                        shape, center_x, center_y, center_z
```

Comment:
This file was temporarily copied before to the new dir to run it with `nose2` in CI. 
This is why there are some apparent changes to it. In fact, this is just a copy of the latest version for this file.